### ISSUE TITLE
Lowering resolution can hide important GUI

### ISSUE THREAD

**Issue by Skasi** [Germanunkol/trAInsported#116]
When starting the tutorial, then going back to menu and lowering the game resolution from, say 1680x1050 to 1024x768 without restarting the game some essential GUI can be lost. In this example the **print("Hello trAIns")** code from tutorial1 can not be seen:
![trains](https://cloud.githubusercontent.com/assets/132906/10262216/341249fc-69be-11e5-9c8c-db88f94fbad5.png)

When increasing the resolution I can see it partially.
![trains](https://cloud.githubusercontent.com/assets/132906/10262226/9bc94852-69be-11e5-8016-84890729a344.png)

**Comment by cmptutr** [comment]
Just restart the game

**Comment by Germanunkol** [comment]
Yes, restarting the game is one solution. Still, this should not happen. I'll look into it, thanks for the bug-report!
